sdks/python/setup.py

```diff
@@ -17,29 +17,42 @@
 
 """Apache Beam SDK setup configuration."""
 
-import os
 import platform
-import re
 import setuptools
+import re
+from apache_beam.version import get_version_from_pom
 
 
-# Currently all compiled modules are optional  (for performance only).
+# Currently all compiled modules are optional (for performance only).
 if platform.system() == 'Windows':
-  # Windows doesn't always provide int64_t.
-  cythonize = lambda *args, **kwargs: []
-else:
-  try:
-    # pylint: disable=g-statement-before-imports,g-import-not-at-top
-    from Cython.Build import cythonize
-  except ImportError:
+    # Windows doesn't always provide int64_t.
     cythonize = lambda *args, **kwargs: []
+else:
```

Comment:
Sorry for the nit but the entire code base uses 2 spaces for indentation. I know that externally 4 spaces are used but we should keep it at 2 until we decide the change the entire code base to 4. So can you please keep it at 2?